dist/index.d.ts

```diff
@@ -25,7 +25,11 @@ export type OrderBy =
   | { [key: string]: Direction }
 
 export type SortKey =
-  | { column: string; direction: Direction }[]
+  | {
+      column: string
+      key?: (table?: string) => string
```

Comment:
Do you need to pass both a key and a column? Or are they actually exclusive? If so I'd maybe add another type to this union to capture that. 

Also I would suggest naming this `sqlExpr` to keep it in line with the rest of join monster. I think there's precedent for `column` or whatever being quoted and `sqlExpr` not being quoted.

Reply:
You need to always pass a column, since it gets added to the selections list, but the key is optional. Do you think it's still worth a separate type in the union?

:+1: for the `sqlExpr` rename.

Reply:
Why does it get added to the selections list? For computing cursors later? If that's the case, the cursor needs to be computed using the `sqlExpr` i think, and not just the column name.